src/doc/users-guide/install.md

```diff
@@ -5,78 +5,86 @@ eleventyNavigation:
-### Linux
- 
- - [{{ download.x86.shell }}]({{ download.host }}releases/download/{{ download.release }}/{{ download.x86.shell }})
-   ({{ download.x86.size }})
- - <a id="{{ download.release }} x86" href="{{ download.host }}releases/download/{{ download.release }}/{{ download.x86.binary }}">{{ download.x86.binary }}</a>
-   ({{ download.x86.size }})
-   
 {% alert "info" %}
-***Note for non-macOS Linux systems:*** You need to install GCC (or equivalent) to provide
-necessary system include files and system libraries.
+***Prerequisites:*** The OpenCilk binaries assume that system header files and system libraries
+are already installed on the system.
+- On Linux, we recommend installing GCC (or equivalent) to provide these necessary system files.
+- On macOS, you must install [XCode](https://developer.apple.com/support/xcode/) or
+the [XCode Command Line Tools](https://mac.install.guide/commandlinetools/index.html) to get
+the necessary system files. 
 {% endalert %}
 
-### macOS
+## Methods of installation
 
- - [{{ download.arm.shell }}]({{ download.host }}releases/download/{{ download.release }}/{{ download.arm.shell }})
-   ({{ download.arm.size }})
- - <a id="{{ download.release }} arm" href="{{ download.host }}releases/download/{{ download.release }}/{{ download.arm.binary }}">{{ download.arm.binary }}</a>
-   ({{ download.arm.size }})
+Precompiled binaries for OpenCilk 2.0 can be installed in several ways:
+- Using a [shell archive (`.sh`)](#installing-using-a-shell-archive) built for the target hardware and operating system.
+- Using a [tarball (`.tar.gz`)](#installing-using-a-tarball) built for the target hardware and operating system.
+- Using a [docker image](#docker-image).
 
-{% alert "info" %}
-***Note for macOS users:*** Unless you are using the OpenCilk Docker image,
-you must also install [XCode](https://developer.apple.com/support/xcode/) or
-the [XCode Command Line
-Tools](https://mac.install.guide/commandlinetools/index.html), which provide
-standard system libraries and header files needed by the OpenCilk compiler.
-{% endalert %}
+You can also [build OpenCilk from source](../build-opencilk-from-source),
+which is the recommended approach for Ubuntu 18.04 and other operating systems for
+which precompiled binaries are not available.
 
-### Docker
+## Installing using a shell archive
 
- - <a id="{{ download.release }} docker" href="{{ download.host }}releases/download/{{ download.release }}/{{ download.docker.binary }}">{{ download.docker.binary }}</a>
-   ({{ download.docker.size }})
+To install OpenCilk 2.0 using a shell archive, first download the appropriate shell
+archive for your system, then follow the installation instructions.
+
+### Download
 
-## Install
+Download the appropriate shell archive for your system using one of the following links:
 
-### Shell archive
+***Linux:***
+ - [{{ download.ubuntu_2004_x86.shell }}]({{ download.host }}releases/download/{{ download.release }}/{{ download.ubuntu_2004_x86.shell }})
```

Comment:
Because it seems so likely I'm mis-typing something, I'm going to do this once, and hope this conveys the idea. Let me know if you want to discuss, or if you know a better way to interface with GA4 and Google Tags.
```suggestion
 - <a id="{{ download.release }} x86" href="{{ download.host }}releases/download/{{ download.release }}/{{ download.ubuntu_2004_x86.shell }}">[{{ download.ubuntu_2004_x86.shell }}</a>
```

Reply:
What happens if multiple distinct download links (specifically, distinct href's) use the same ID?

My concern is that we don't distribute, for instance, a _single_ x86 binary; we have multiple.  In particular, we have x86_64 binaries for two different versions of Ubuntu, and we have x86_64 binaries for x86_64 macOS.  Is it OK if all of those distinct download links share the same ID?

Reply:
Good question @neboat. I am reviewing my research on Google Analytics (GA4), and see that we might not need custom `id`s for links after all. So I'd like to hold off on directly answering your question and put this into a bigger question about analytics. I am going to follow up on #20, where all the analytics work is being discussed.

Reply:
My conclusion based on #20 is that we don't need custom `id`s in links after all, and therefore I retract my requested change. That said, your question about granularity still stands, and I'll say more about that in #20.